packages/next-auth/lib/env.ts

```diff
@@ -0,0 +1,20 @@
+import { parseEnv, z } from "znv";
+
+export const env = parseEnv(process.env,
```

Comment:
we gotta use the individual env variables to trigger turbo's env variable cache eslint rule.

```suggestion
export const env = parseEnv({ GITHUB_ID: process.env.GITHUB_ID, GITHUB_SECRET: process.env.GITHUB_SECRET,
```

Reply:
Cizus 🤦‍♂️ I'll revert this